src/sonic-build-hooks/scripts/symlink_build_hooks

```diff
@@ -2,6 +2,7 @@
 
 HOOK_PATH=/usr/local/share/buildinfo/hooks
 TARGET_PATH=/usr/sbin
+[ -d /usr/local/sbin ] && TARGET_PATH=/usr/local/sbin
```

Comment:
>/usr/local/sbin [](start = 5, length = 15)

To simplify the logic, we can assume `/usr/local/sbin` exists and fail if not.

Reply:
Simply change to /usr/local/sbin, we assume the path exists, if not will be failed when creating the symbol link.

---
In reply to: [556191523](https://github.com/Azure/sonic-buildimage/pull/6422#discussion_r556191523) [](ancestors = 556191523)